config/manager/manager.yaml

```diff
@@ -43,7 +66,7 @@ spec:
             - name: certs
               mountPath: /etc/ssl/certs
             - name: credentials
-              mountPath: /etc/aws
+              mountPath: /root/.aws
```

Comment:
This assumes that the controllers run as root. Is that intentional?

Reply:
They do, the docker container user is root (for now).